straxen/contexts.py

```diff
@@ -96,7 +101,7 @@ def xenonnt_online(output_folder='./strax_data',
             st.storage.append(
                 strax.DataDirectory(output_folder))
 
-        st.context_config['forbid_creation_of'] = straxen.daqreader.DAQReader.provides + ('records',)
```

Comment:
Actually since pulse_processing is done with we_are_the_daq, I can keep this as-is, and this will prohibit people from accidentally reprocessing records even with the new settings.